### PR TITLE
Update net/toxcore to 0.1.11 and net/toxic to 0.8.1

### PR DESCRIPTION
**Pull request by dmgk**

### CODE REVIEW

**Comment by dmgk** [comment]
Any feedback on this? I've been running this combo for about a week w/o issues.

**Comment by aSourceFish** [comment]
You've reinvented my versions https://github.com/aSourceFish/myports/tree/master/net

**Comment by dmgk** [comment]
@aSourceFish Your versions are more advanced, I didn't add support for audio and python -)

Anyway, toxcore and toxic seem pretty stable now, it would be good to have them merged and  available as packages.

File: net/toxcore/Makefile [hunk 1]
```diff
@@ -23,7 +23,7 @@ GH_ACCOUNT =		TokTok
 GH_PROJECT =		c-toxcore
 GH_TAGNAME =		v$V
 
-MODULES =		gnu gcc4
+MODULES =		gnu
```
[the remark this answers is not in the document]
Comment (dmgk) [reply net/toxcore/Makefile]: @ibara Thanks, I'm away from OpenBSD box now, will update later today.

[the remark this answers is not in the document]
Comment (ibara) [reply net/toxcore/Makefile]: Also provide a comment above the COMPILER line saying why it's needed.
```
# C++
COMPILER = base-clang ports-gcc ports-clang
```
For example.

**Comment by aSourceFish** [comment]
@dmgk I've removed audio support by default because in OpenBSD snapshot (before new year) openal lib was a complete crap and caused segmentation faults in applications which use it. Also audio input won't work anyway (again, openal is crap), but I'll remove my change about audio, so audiocalls and sound notifications will always be available. Let me send PR

**Comment by aSourceFish** [comment]
@dmgk @ibara https://github.com/jasperla/openbsd-wip/pull/81

**Comment by ibara** [comment]
Closed by #81

**Comment by dmgk** [comment]
Thanks for taking care of that -)

**Comment by aSourceFish** [comment]
@dmgk ты не представляешь как долго я этого ждал)
Если что со мной можно поговорить: 0F09F10750FF5FE90D80290ADCC10CBEB27BCEC449101195DC0852A39AB5674DDDFB58D4D7AB

**Pull request closed without merging**